src/Fmt_ast.ml

```diff
@@ -972,12 +984,8 @@ and fmt_pattern c ?pro ?parens ({ctx= ctx0; ast= pat} as xpat) =
       , Some {ppat_desc= Ppat_tuple [x; y]; ppat_attributes= []} ) -> (
     match sugar_list_pat c pat with
     | Some (loc_xpats, nil_loc) ->
-        let wrap =
-          if c.conf.space_around_collection_expressions then wrap "[ " "@ ]"
-          else wrap_fits_breaks c.conf "[" "]"
-        in
         hvbox 0
-          (wrap
+          (wrap_list c.conf
```

Comment:
Normally we pass `c` in full, and only select out what is needed at the use site. Maybe better to do that here as well.